pkg/brctl/util.go

```diff
@@ -91,52 +91,64 @@ func getIndexFromInterfaceName(ifname string) (int, error) {
 	return int(ifrIfindex), nil
 }
 
-// set values for the bridge
-// all values in the sysfs are of type <bytes> + '\n'
-func setBridgeValue(bridge string, name string, value []byte, _ uint64) error {
-	err := os.WriteFile(filepath.Join(BRCTL_SYS_NET, bridge, "bridge", name), append(value, BRCTL_SYS_SUFFIX), 0)
-	if err != nil {
-		return err
+var ErrBridgeNotExist = errors.New("bridge does not exist")
```

Comment:
I think there are quite a few custom errors scattered around the brctl package. Maybe it does make sense to keep them in one place?

Reply:
I'm planning to also file a PR for integration tests and overall unit test improvement. Will do it there. Just want to get that path set in order to fix functionality.